### PR TITLE
Pv testing bugdfix

### PR DESCRIPTION
**Pull request by M00staff**

### CODE REVIEW

File: src/Components/AoPage/AoPage.jsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const AoPage = () => (
       <Route path="/profile/ao/availablebidders" render={() => <AvailableBidderContainer isCDO={false} isAO />} />
       <Route path="/profile/ao/panelmeetingagendas/:pmID" render={() => <PanelMeetingAgendas isAO />} />
       <Route path="/profile/ao/cyclemanagement" render={() => <CycleManagement isAO />} />
-      <Route path="/profile/ao/projectedvacancy" render={() => <ProjectedVacancy isCDO={false} isAO />} />
+      <Route path="/profile/ao/projectedvacancy" render={() => <ProjectedVacancy isCDO={false} isAO viewType="ao" />} />
```
Comment (jlam-mpc) [inline src/Components/AoPage/AoPage.jsx:29]: I think we should scrap the isAO prop since you have this new one

Comment (jlam-mpc) [reply src/Components/AoPage/AoPage.jsx:29]: I removed that prop in the PV components but forgot to remove it here

You new prop is a lot more descriptive than isAO, but I think the isAO prop can handle what you're aiming to do as well if you just pull that into the ProjectedVacancy components. Just leave it and delete "isAO"

Comment (M00staff) [reply src/Components/AoPage/AoPage.jsx:29]: wow - just realized it literally wasn't doing anything cause this guy 

image

Comment (M00staff) [reply src/Components/AoPage/AoPage.jsx]: also dont need the `isCDO={false}` bit either - not used lol

**Review by jlam-mpc — APPROVED** [review]

**Pull request merged**